src/quo2/components/banners/banner/style.cljs

```diff
@@ -3,17 +3,26 @@
 
 
 (def container
-  {:width              "100%"
-   :height             50
-   :background-color   colors/primary-50-opa-20
-   :flex-direction     :row
-   :align-items        :center
-   :padding-horizontal 20
-   :padding-vertical   10})
+  {:flex             1
+   :height           40
+   :background-color colors/primary-50-opa-20
+   :flex-direction   :row
+   :align-items      :center
+   :padding-right    22
+   :padding-left     20
+   :padding-vertical 10})
 
 (def counter
-  {:padding-right   22
-   :height          20
-   :width           20
+  {:height          20
+   :position        :absolute
+   :right           22
+   :flex            1
    :justify-content :center
-   :align-items     :center})
\ No newline at end of file
+   :align-items     :center})
+
+(def icon {:margin-right 8})
+
+(defn text
+  [hide-pin?]
+  {:width (if hide-pin? 307 279)})
```

Comment:
please don't use width it should be flexible

Reply:
sure, will adjust!